### ISSUE TITLE
Impossible to translate

### ISSUE THREAD

**Issue by g964** [Codewars/codewars.com#1090]
Kata: <https://preview.codewars.com/kata/phone-directory/translations>
When I push the "Translate this kata" I get
```
Looks like there was a server error! (500)

We have taken note of the situation and will be looking into it.

If you would like to supply us with some details as to how this happened please visit our Github issues list. This will help us get things fixed faster.

undefined method `name' for nil:NilClass
```

**Comment by Voileexperiments** [comment]
I can open the translation window just fine (on both main/preview site).

Is it something user-specific again?

**Comment by g964** [comment]
Same error with main and preview! Impossible to work a bit! Maybe it is @g964 specific:-)

**Comment by jhoffner** [comment]
@g964 is it just this translation or any translation?

**Comment by g964** [comment]
I only try this other one: <http://www.codewars.com/kata/rainfall/translations> and it works 

BUT the first for which I posted: <https://preview.codewars.com/kata/phone-directory/translations> doesn't work; always the same server error.

**Comment by g964** [comment]
Still unable to translate...

**Comment by g964** [comment]
Kata: https://preview.codewars.com/kata/phone-directory/translations

Still unable to translate... First post: 14 days ago:-(
```
Looks like there was a server error! (500)

We have taken note of the situation and will be looking into it.

If you would like to supply us with some details as to how this happened please visit our Github issues list. This will help us get things fixed faster.

undefined method `name' for nil:NilClass
```

**Comment by jamad** [comment]
I have reported #1224 that could be the duplicate of this issue.
I have got 3 translations as contribution but because of this issue, I cannot take any action for the translator.
All 3 links (https://www.codewars.com/kata/sequence-to-1/discuss#5a3c0b71f28b8272e300002b)  took me to the server error page.
By the way, the issue doesn't affect the translation for another kata I made. So it seems to be the specific kata issue only for the kata creator. Others seems to be able to see the kumite page.
